packages/api/src/conversation-subgroup/index.ts

```diff
@@ -1,28 +1,28 @@
-import { SDNAClass, SubjectEntity, SubjectFlag, SubjectProperty, LinkQuery, Literal } from "@coasys/ad4m";
+import { ModelOptions, Ad4mModel, Flag, Property, LinkQuery, Literal } from "@coasys/ad4m";
 import Topic, { TopicWithRelevance } from "../topic";
 import SemanticRelationship from "../semantic-relationship";
 import Conversation from "../conversation";
 import { SynergyTopic, SynergyItem, icons } from "@coasys/flux-utils";
 
-@SDNAClass({
+@ModelOptions({
   name: "ConversationSubgroup",
 })
-export default class ConversationSubgroup extends SubjectEntity {
-  @SubjectFlag({
+export default class ConversationSubgroup extends Ad4mModel {
+  @Flag({
     through: "flux://entry_type",
     value: "flux://conversation_subgroup",
   })
   type: string;
 
-  @SubjectProperty({
+  @Property({
     through: "flux://has_name",
     writable: true,
     resolveLanguage: "literal",
     required: false,
```

Comment:
Could use `Optional` instead of `required: false`
```suggestion
  @Optional({
    through: "flux://has_name",
    writable: true,
    resolveLanguage: "literal",
```